composer/rest/README.rst

```diff
@@ -61,14 +61,37 @@ Install Dependencies
 Samples
 -------------------------------------------------------------------------------
 
-Determine Cloud Storage path for DAGs
+Determine client ID associated with a Cloud Composer environment
 +++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-.. image:: https://gstatic.com/cloudssh/images/open-btn.png
-   :target: https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=composer/rest/get_dag_prefix.py,composer/rest/README.rst
+To run this sample:
 
+.. code-block:: bash
 
+    $ python get_client_id.py
+    usage: get_client_id.py [-h] project_id location composer_environment
 
+    Get the client ID associated with a Cloud Composer environment.
+
+    positional arguments:
+      project_id            Your Project ID.
+      location              Region of the Cloud Composer environment.
+      composer_environment  Name of the Cloud Composer environment.
+
+    optional arguments:
+      -h, --help            show this help message and exit
+
+In case of any issue with `get_client_id.py`_, you can also get the client ID using the following command, where `AIRFLOW_URL`_ would be the URL of your Airflow webserver (ex. `https://*****************-tp.appspot.com`_):
```

Comment:
nit: extra underscore ("_") after `get_client_id.py` and `AIRFLOW_URL`

Reply:
and `https://*****************-tp.appspot.com`

Reply:
No problem - thanks @dandhlee !